### ISSUE TITLE
Does anybody know why the score jump a lot when dropping the learning rate last few epoches during the training?

### ISSUE THREAD

**Issue by AI-Passionner** [IDEACVR/DINO#92]
@SuperHenry2333 I only see this behavior in YOLOX when turning off the augmentation in the last few epochs. Most of the other object detections didn't show this behavior. In other words, why didn't you try the cosine learning rate scheduler? 

In my case, when the learning rate dropped at the last epoch, the score dramatically gained 3%, which is very solid.

**Comment by SuperHenry2333** [comment]
We adopted the same learning rate schedule as previous DETR-like works to suddenly drop to 1/10 of the original learning rate. We did not compare sudden drop with cosine drop. Methods adopting sudden drop usually have a large performance gain when dropping the learning rate.
